lib/builder.js

```diff
@@ -247,36 +449,51 @@ function visitTree(tree, moduleName, pluginLoader, visit, seen) {
   });
 }
 
-// returns a new tree containing tree1 n tree2
-Builder.prototype.intersectTrees = function(tree1, tree2) {
-  var name;
-  var intersectTree = {};
+
+// takes 2-N arguments (tree1, tree2, treeN), returns a new tree containing tree1 n tree2 n treeN
+Builder.prototype.intersectTrees = function() {
```

Comment:
Can you rather construct your own intersection reduction function than overloading this one?